dashboard/app/controllers/aichat_controller.rb

```diff
@@ -26,6 +30,37 @@ def chat_completion
     render(status: :ok, json: response_body)
   end
 
+  # params are newAichatEvent: AichatEvent, aichatContext: {currentLevelId: number; scriptId: number; channelId: string;}
+  # POST /aichat/log_aichat_event
```

Comment:
nit: maybe just `/aichat/log_event` since we already have `aichat` in the path?

Reply:
What do you think of `aichat/log_chat_event`?

Reply:
Will go with `aichat/log_chat_event` since we renamed events on the front-end `ChatEvent`. Thanks!